addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -247,9 +256,15 @@ const LinkPopoverWidget = Widget.extend({
      * @param {Event} ev
      */
     _onRemoveLinkClick(ev) {
+        const content = this.$target[0].children[0];
         ev.preventDefault();
         this.options.wysiwyg.odooEditor.execCommand('unlink');
         ev.stopImmediatePropagation();
+        if (isImg(content)) {
+            // Deselect and reselect.
+            content.parentElement.click();
+            content.click();
```

Comment:
In general it is better not to rely on click() or any other event... Let's imagine that later on we decide that it is no longer click event that selects but mousedown... this will not be robust enough... Can you double check if there is not another (more robust) way to do this?

Reply:
Double checked.
Did not manage to get a similar effect neither through `trigger_up()` nor `bus.trigger()` using various events: it led to an incompletely selected state, causing further unexpected behaviors.

Reply:
So activate_snippet does not work? :thinking:

Reply:
It works in terms of updating the options form - and so does `snippet_option_update`, but the reached selection state is not fully correct: if you press CTRL+K it leads to incorrect transformation of the DOM.

Reply:
Added an event on ReplaceMedia to deactivate. This makes the option focused - which makes CTRL+K handled at browser window level instead - at least not corrupting the DOM.